core/arch/arch.h

```diff
@@ -434,7 +434,11 @@ typedef struct _clean_call_info_t {
     bool skip_save_flags;
     bool skip_clear_flags;
     int num_simd_skip;
+#if defined(AARCH64)
+    bool simd_skip[MCXT_NUM_SIMD_SLOTS + MCXT_NUM_SVEP_SLOTS + MCXT_NUM_FFR_SLOTS];
```

Comment:
See the comments above about a helper/define.  Plus: should MCXT_NUM_SIMD_SLOTS include the SVE ones?  It must include AVX-512 on x86, so separating them just for AArch64 may cause confusion and complexity.

Reply:
See https://github.com/DynamoRIO/dynamorio/pull/5835#discussion_r1247192964.